specs/lightclient/lightclient-fork.md

```diff
@@ -0,0 +1,83 @@
+# Ethereum 2.0 Light Client Support -- From Phase 0 to Light Client Patch
+
+**Notice**: This document is a work-in-progress for researchers and implementers.
+
+## Table of contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Configuration](#configuration)
+- [Fork to Light-client patch](#fork-to-light-client-patch)
```

Comment:
We write "light client" in three different ways  😂 

1) light client
2) lightclient
3) light-client

I guess we want to be consistent. (Small preference for "light client" on my side.)

Reply:
👍 on "light client".

Once we have a better hard fork name, we can replace the folder name `lightclient` with it.
We can scan it again after merging this PR to avoid conflicts.